### PR TITLE
Add publish task for grade to publish ballerina-openapi artifact into maven

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Add publish task for grade to publish ballerina-openapi artifact into maven

### CODE REVIEW

File: ballerina-to-openapi/build.gradle [hunk 1]
```diff
@@ -132,3 +132,27 @@ compileJava {
         classpath = files()
     }
 }
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
```
Comment (lnash94) [inline ballerina-to-openapi/build.gradle:138]: will this publish to GitHub packages or Is there a reason not to publish in GitHub packages also?

Comment (SasinduDilshara) [reply ballerina-to-openapi/build.gradle:138]:
```
publishing {
    publications {
        mavenJava(MavenPublication) {
            artifact source: createArtifactZip, extension: 'zip'
        }
    }

    repositories {
        maven {
            name = "GitHubPackages"
            url = uri("https://maven.pkg.github.com/ballerina-platform/openapi-tools")
            credentials {
                username = System.getenv("packageUser")
                password = System.getenv("packagePAT")
            }
        }
    }
}
```

This is how `module-ballerina-openapi` got published, Don't I need to do the same in here

File: ballerina-to-openapi/build.gradle [hunk 2]
```diff
@@ -132,3 +132,27 @@ compileJava {
...
+            groupId project.group
```
Comment (TharmiganK) [inline ballerina-to-openapi/build.gradle:139]: Can we change the group here? The project group is `io.ballerina`. I think it is better to publish it under `io.ballerina.openapi`
```suggestion
            groupId "io.ballerina.openapi"
```

File: ballerina-to-openapi/build.gradle (hunk 1 above)
Comment (TharmiganK) [reply ballerina-to-openapi/build.gradle:138]: I think this should work

Comment (TharmiganK) [reply ballerina-to-openapi/build.gradle:138]: I think this is correct

File: ballerina-to-openapi/build.gradle (hunk 2 above)
Comment (SasinduDilshara) [reply ballerina-to-openapi/build.gradle:139]: `Persist tool` is publish to `io.ballerina`.
https://github.com/ballerina-platform/persist-tools/blob/a487cbc0c085e572d042a74e851d1a2e51e4da3a/persist-cli/build.gradle#L151

Don't we need to have a same group id among the tools?

Comment (TharmiganK) [reply ballerina-to-openapi/build.gradle:139]: That is the entire tool right? This is just a part of the tool

Comment (SasinduDilshara) [reply ballerina-to-openapi/build.gradle:139]: ok, will update that

**Review by TharmiganK — APPROVED** [review]
LGTM

**Pull request merged**